susemanager-utils/susemanager-sls/salt/uptime-tracker/suse-upstream-tracker.sls

```diff
@@ -0,0 +1,35 @@
+{%- if not salt['pillar.get']('susemanager:distupgrade:dryrun', False) %}
```

Comment:
I think you do not add it to the distupgrade state.
I think you do not need this "if" condition.

Reply:
Thank you. I am going to pull everything but dump-uptime-info from the file. The other items were really just to make my testing easier. Ultimately installation of the package and enabling the collection will be done by the install of the base OS.

Reply:
Cleaned unneeded code from the file. I left os_family Suse check, because we only need to check for the file on SUSE based systems. I did not know if having the checked kept the server from having the action run on the client or not. If the action is run on the client anyway, then that check can also be removed.